package.json

```diff
@@ -11,6 +11,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
+    "styled-components": "^5.3.5",
```

Comment:
Emotion is better than styled components and does the same
https://emotion.sh/docs/styled

Is the library I mentioned to you at the beginning to use